pict-doc/pict/scribblings/pict.scrbl

```diff
@@ -543,7 +543,7 @@ argument for consistency with the other functions.}
                       [#:style style (one-of/c 'transparent 'solid 'xor 'hilite 
                                                'dot 'long-dash 'short-dash 'dot-dash 
                                                'xor-dot 'xor-long-dash 'xor-short-dash 
-                                               'xor-dot-dash)]
+                                               'xor-dot-dash) 'solid]
```

Comment:
Same for `pin-arrow-line`, and the examples for both don't give `#:style` arguments but make solid lines. [docs](http://docs.racket-lang.org/pict/Basic_Pict_Constructors.html?q=add-line#%28def._%28%28lib._pict%2Fmain..rkt%29._pin-arrows-line%29%29)

But let's just change the docs to match the functions actual contract (which allows `#f`).